app/Services/Eggs/Sharing/EggImporterService.php

```diff
@@ -25,6 +26,18 @@ public function handle(UploadedFile $file): Egg
     {
         $parsed = $this->parser->handle($file);
 
+        $replacements = [
```

Comment:
This should be placed inside a `const`, `app/Services/Eggs/Sharing/EggUpdateImporterService.php` uses the same exact same array

Reply:
I mean... It works <3

Reply:
It should be only in the code once